test/tests/concrete_moisture_heat_transfer/moisture_diffusion_validation_base.i

```diff
@@ -0,0 +1,212 @@
+[Mesh]
```

Comment:
What does this validate against? Please add comments. 

Also, please add verification tests for all the different model options/combinations.

Reply:
I added test cases for possible combinations.